src/containers/tutor-home-page/language-step/LanguageStep.jsx

```diff
@@ -1,19 +1,116 @@
+import React, { useState } from 'react'
 import Box from '@mui/material/Box'
-
-import { styles } from '~/containers/tutor-home-page/language-step/LanguageStep.styles'
+import Typography from '@mui/material/Typography'
+import TextField from '@mui/material/TextField'
+import Autocomplete from '@mui/material/Autocomplete'
+import { styled } from '@mui/material/styles'
 import img from '~/assets/img/tutor-home-page/become-tutor/languages.svg'
+import { styles } from '~/containers/tutor-home-page/language-step/LanguageStep.styles'
+
+const languages = [
+  'English',
+  'Ukrainian',
+  'Polish',
+  'German',
+  'French',
+  'Spanish',
+  'Arabic'
+]
+
+const Container = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  padding: '16px',
+  [theme.breakpoints.down('md')]: {
+    flexDirection: 'column',
+    alignItems: 'center'
+  }
+}))
+
+const ImgContainer = styled(Box)(({ theme }) => ({
+  flex: 1,
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  [theme.breakpoints.down('md')]: {
+    display: 'none'
+  }
+}))
+
+const Img = styled('img')(({ theme }) => ({
+  width: '100%',
+  maxWidth: '400px',
+  [theme.breakpoints.down('md')]: {
+    maxWidth: '300px'
+  },
+  [theme.breakpoints.down('sm')]: {
+    maxWidth: '100%',
+    marginBottom: '16px'
+  }
+}))
+
+const RightBox = styled(Box)(({ theme }) => ({
+  flex: 2,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  //padding: '16px',
+  [theme.breakpoints.down('md')]: {
+    padding: '0'
+  }
+}))
+
+const Title = styled(Typography)(({ theme }) => ({
+  marginBottom: '16px',
+  [theme.breakpoints.down('sm')]: {
+    marginBottom: '8px'
+  }
+}))
+
+const AutocompleteStyled = styled(Autocomplete)({
+  width: '100%',
+  marginBottom: '16px'
+})
+
+const MobileImgContainer = styled(Box)(({ theme }) => ({
+  display: 'none',
+  [theme.breakpoints.down('sm')]: {
+    display: 'block',
+    textAlign: 'center'
+  }
+}))
```

Comment:
I like the styled components approach, but move this code to another file to make the `LanguageStep.jsx` shorter.